nh_odoo_fixes/remove_exception_name_from_error_dialogs.py

```diff
@@ -0,0 +1,15 @@
+from openerp.osv.orm import except_orm
+
+
+def new_init(self, name, value):
```

Comment:
Would it not be better to call the original `__init__` method then just change `self.args` instead of redefining it? This way we still get any changes upstream if we move to newer Odoo version